tools/privacyidea-diag

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 #
-# 2018-07-31 Cornelius Kölbel <cornelius.koelbel@netknights.it>
+# 2018-07-31 Cornelius Koelbel <cornelius.koelbel@netknights.it>
```

Comment:
Isn't there a way to use utf8 anyways - like adding a header like in a python script?
I like my name! ;-)

Reply:
Doesn't seem so :-( But I just found https://github.com/pypa/setuptools/pull/1389 which is super-fresh and seems to fix this! So maybe we can also just wait for the next pip version ...

Reply:
Waiting is lame :-/